tests/kafkatest/services/security/security_config.py

```diff
@@ -23,20 +24,21 @@
 
 class SslStores(object):
     def __init__(self):
-        self.ca_crt_path = "/tmp/test.ca.crt"
-        self.ca_jks_path = "/tmp/test.ca.jks"
+        self.ca_dir = mkdtemp(dir="/tmp")
+        self.ca_crt_path = os.path.join(self.ca_dir, "test.ca.crt")
+        self.ca_jks_path = os.path.join(self.ca_dir, "test.ca.jks")
         self.ca_passwd = "test-ca-passwd"
 
-        self.truststore_path = "/tmp/test.truststore.jks"
+        self.truststore_dir = mkdtemp(dir="/tmp")
```

Comment:
I think truststore and ca could be stored in a single directory. The files are related and are created together.